cmd/antrea-agent/options.go

```diff
@@ -192,17 +192,45 @@ func (o *Options) setDefaults() {
 func (o *Options) validateFlowExporterConfig() error {
```

Comment:
I am having trouble understanding why we don't inject the default target host (`FlowAggregatorDNSName`) in this code, like we typically do for default values? IMO it has the potential to simplify the code and perhaps remove the `goto`

Reply:
Good suggestion. We could simplify the code. Refactored the code.